Update configuring-scim-with-azure-ad.md (#5253)
postmanlabs/postman-docs: src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md

To set up provisioning with Microsoft Entra ID, do the following:

1. In Postman, [enable SCIM](/docs/administration/scim-provisioning/scim-provisioning-overview/#enabling-scim-in-postman) and [generate a SCIM API key](/docs/administration/scim-provisioning/scim-provisioning-overview/#generating-scim-api-key).

> 1. Sign in to the [Microsoft Entra ID portal](https://entra.microsoft.com/).
> 1. Select **Applications**, and then **Enterprise applications** from the left pane.
> 1. Select the Postman app if you have already added the same for configuring SSO. Otherwise, select **+ New application**, search for "Postman" and select the Postman app from the results, then select Create.

Reviewer: ```suggestion
1. Sign in to the [Microsoft Entra ID portal](https://entra.microsoft.com/).
1. Select **Applications**, then select **Enterprise applications** in the left pane.
1. Select the Postman app if you've already configured [SSO with Microsoft Entra ID](/docs/administration/sso/azure-ad/). Otherwise, select **+ New application**, search for "Postman", select the Postman app from the results, then select **Create**.
```
